### DISCUSSION TITLE
Venus rebranding / `Venus`品牌重塑｜产品架构调整｜讨论

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#5420] [Ideas]
## 背景

相较于`filecoin`参考实现的`data-onboarding`组合拳`Lotus`+`Boost`，当前`Venus`采用了组件化和模块化的设计，使得`Venus`当前泛指了很多不同的`Venus`组件/模块。这个情况造成了用户在理解各组件关系和功能时有困难；外部生态团队/PL团队对`Venus`认知上存在障碍。

### 当前`Venus`品牌结构

当前`Venus`按功能结构划分为链服务，算力服务，订单服务。各服务依次分别对应了不同组件，链服务对应了`venus`, `venus-auth`, `venus-messager`, `venus-miner`, `venus-gateway`组件；算力服务对应了`venus-sector-manager`, `venus-worker`, `venus-wallet`组件；订单服务对应了`venus-market`, `market-client`, `venus-wallet`组件。

类似`Lotus`需要和`Boost`，`Indexer`，等等进行协同开发一样，`Venus`各服务之间需要良好的配合，但同时也有着相对独立的功能/业务需求和使用场景。在[venus-docs](https://venus.filecoin.io/zh/)上，对每个服务也进行了区分。

### 外部对`Venus`产品认识

1. 社区，SP等。社区新用户，或者从`Lotus`体系转移过来的用户，对`venus`当前众多组件体系对应的各组件功能也不能很好的一一对应上。现已有一层产品抽象，即，链服务，算力服务，订单服务。但3者没有很好的形成品牌。

2. 外部生态团队。在和外部生态团队的合作过程中，和社区新用户，SP面临类似问题。让生态团队更快的理解`venus`，基于`Venus`合作/开发，需要一个更好的产品功能和类型的划分。

3. PL团队/Grant团队。在和`grant`团队的接触过程中，发现当需要对`venus`更进一步添加功能改造时，因为当前`venus`品牌结构泛指所有不同功能`venus`组件时，`grant`团队不能有效的理解新功能改造需求。造成PL团队无法理解团队在开发上，一些组件和架构想法对生态基建的帮助。

## 品牌升级

基于上述的一些问题，本提案对`venus`进行产品架构的品牌调整。根据`venus`各组件定位，功能等等，包装出如下3个品牌。对应Venus的链服务，算力服务，订单服务。分别命名为`Sophon`, `Damocles`, `Droplet`。

### 链服务

https://venus.filecoin.io/zh/cs
作为`venus`分布式体系的核心，像`sophon`一样组织基建。

### 算力服务

https://venus.filecoin.io/zh/cluster
增长和维持算力的组件。灵活部署，完全支持SaaS。蕴含巨大的力量。

### 订单服务

https://venus.filecoin.io/zh/market/
支持`filecoin`存储订单流程的组件。包括接单和发单等等。

### 定位

`Venus`由于其长期的品牌运营，已在用户心中有一个扎实的形象。`Venus`在品牌重塑后，还是泛指所有`venus`体系下的所有组件。面对SP知识体系相对健全的用户，当前链服务，订单服务，算力服务的3个描述，已经能够让用户对`venus`有一定的认知。基于这3个描述，对新用户更抽象的一层品牌即为当前品牌重塑的重点。ie，`Apollo`, `Hercules`, `Hermes`。

### 待交付 deliverables

- 各品牌全新`名称`：一个让人容易记住，中内外都可适用的品牌名称（中英文），方便写入子域名的；
- 各品牌全新`Logo`：使用户对品牌有直观的视觉印象；
- 各品牌功能服务简介`one-pager`：对品牌功能的单页快速描述；
- 各品牌自有的文档，以及网站（xxx.filecoin.io，可能需要和PL申请子域名）：参考 boost.filecoin.io

### 名称参考

- 链服务 -> [Apollo](https://en.wikipedia.org/wiki/List_of_Greek_mythological_figures)；算力服务 -> Hercules；订单服务 -> Hermes；
- (备选) 链服务 -> Nexus；算力服务 -> Colossus；订单服务 -> Pheonix；

### 目标

目标22年底圣诞至元旦期间发布。

### milestone

距离目标大概4个sprint。安排推进计划如下。（2022年11月4日制定）

- Logo（1-2个sprint）: png, svg, 大，小，icon；能在slack，等较小的格式下（emoji的形式），给用户留下印象；和品牌新名称（sophon 智子，damocles 执剑人，droplet 水滴）有关联；和`venus`原有logo有一定关联；
- `one-pager`（1个sprint）: 结合各组件新的发展，配合生态新的措辞，对3个品牌新的单页描述；
- 品牌文档页（2个sprint）：各个品牌形成各自的文档页面；与PL沟通，获取各品牌的子域名；venus.filecoin.io 作为主要入口，引导至3个品牌的子页域名页面；

**Comment by 0x5459** [comment]
`Apollo` 这个名字可能有点太常见了，github上面能搜到一堆

**Comment by Fatman13** [comment]
看了几个三体里面武器的名称：

链服务：智子 -> Sophon；三体星人进攻地球的中枢
算力服务：执剑人 - > Damocles
订单服务：水滴 -> Droplet

二向箔 -> two-vector foil；有点太长了，虽然非常厉害。。。

**Reply by 0x5459** [reply]
不要回答！不要回答！不要回答！

**Comment by Fatman13** [comment]
## 新Logo预览

智子：
![image](https://user-images.githubusercontent.com/1591330/205581370-d467d776-60a4-4b37-b25a-58fa82adb156.png)

执剑人：
![image](https://user-images.githubusercontent.com/1591330/205581532-f0073513-7f52-4c1e-ad98-096592e58bdc.png)

水滴：
![image](https://user-images.githubusercontent.com/1591330/205581638-e2955fa7-0da4-42fa-82d0-debd00b3f368.png)
